### ISSUE TITLE
tooltip position

### ISSUE THREAD

**Issue by halpz** [JoeFryer/JDFTooltips#11]
is there a setting to position the tooltip (arrow facing down) with the tooltip centred above the view and not to the side?

**Comment by JoeFryer** [comment]
No, there isn't at the moment, although this might be nice to have. I'll leave this issue open and perhaps get around to it soon. Or, PRs welcome.
